src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -68,7 +69,11 @@ public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFact
         {
             var logger = _loggerFactory.GetOrCreate(FunctionsExecutionEventsCategory);
             string currentUtcTime = DateTime.UtcNow.ToString();
-            string log = string.Join(",", executionId, siteName, concurrency.ToString(), functionName, invocationId, executionStage, executionTimeSpan.ToString(), success.ToString(), currentUtcTime);
+            string log = currentUtcTime;
+            if (FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableLinuxEPExecutionCount))
+            {
+                log = string.Join(",", executionId, siteName, concurrency.ToString(), functionName, invocationId, executionStage, executionTimeSpan.ToString(), success.ToString(), currentUtcTime);
+            }
             WriteEvent(logger, log);
```

Comment:
I think this might be cleaner since log is just getting assigned to another string on 72, we really only need the log string if the feature flag is on

```suggestion
            if (FeatureFlags.IsEnabled(ScriptConstants.FeatureFlagEnableLinuxEPExecutionCount))
            {
                string log = string.Join(",", executionId, siteName, concurrency.ToString(), functionName, invocationId, executionStage, executionTimeSpan.ToString(), success.ToString(), currentUtcTime);
                WriteEvent(logger, log);
                return;
            }

            WriteEvent(logger, currentUtcTime);
```

Reply:
Or else-statement instead of return - either way